### ISSUE TITLE
words get lost in wordcloud

### ISSUE THREAD

**Issue by nofonbg** [mebis-lp/mod_mootimeter#69]
when clicking button showresults repeatedly sometimes words disappear
![Strip 1](https://github.com/mebis-lp/mod_mootimeter/assets/118652526/465173ff-96ef-4b58-a47d-90ad3eab9f47)

**Comment by nofonbg** [comment]
Yes they randomly apper and dissapear (don't get lost in the db) like i n the screenshots (left "pizza", right "Kaiserschmarrn" ist missing)

**Comment by PM84** [comment]
Yes. this is a known issue of the used library: https://github.com/timdream/wordcloud2.js

**Comment by PM84** [comment]
Will be solved with: https://github.com/mebis-lp/mod_mootimeter/issues/44
